### PR TITLE
Move the updated transaction structure from ZIP 226 to ZIP 230

### PR DESCRIPTION
**Pull request by vivek-arte**
This PR removes the transaction structure fields from ZIP 226 in favour of adding them to ZIP 230, for the v6 Transaction Format.

### CODE REVIEW

File: zip-0230.rst [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``192 * nOutputsSapling``           |``vOutputProofsSapling``  |``byte[192 * nOutputsSapling]``         |Encodings of the zk-SNARK proofs for each Sapling Output.                  |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``64``                              |``bindingSigSapling``     |``byte[64]``                            |A Sapling binding signature on the SIGHASH transaction hash.               |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+| **Orchard Transaction Fields**                                                                                                                                                     |
```
Comment (PaulLaux) [inline zip-0230.rst:143]: Orchard-ZSA Transaction Fields

File: zip-0230.rst [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``varies``                          |``nActionsOrchard``       |``compactSize``                         |The number of Orchard Action descriptions in                               |
+|                                    |                          |                                        |``vActionsOrchard``.                                                       |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``852 * nActionsOrchard``           |``vActionsOrchard``       |``ZSAOrchardAction[nActionsOrchard]``   |A sequence of ZSA Orchard Action descriptions, encoded per                 |
+|                                    |                          |                                        |the `ZSA Orchard Action Description Encoding`.                             |
```
Comment (PaulLaux) [inline zip-0230.rst:149]: Orchard-ZSA Action ...

File: zip-0230.rst [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+| **Orchard Transaction Fields**                                                                                                                                                     |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``varies``                          |``nActionsOrchard``       |``compactSize``                         |The number of Orchard Action descriptions in                               |
+|                                    |                          |                                        |``vActionsOrchard``.                                                       |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``852 * nActionsOrchard``           |``vActionsOrchard``       |``ZSAOrchardAction[nActionsOrchard]``   |A sequence of ZSA Orchard Action descriptions, encoded per                 |
```
Comment (PaulLaux) [inline zip-0230.rst:148]: **OrchardZsaAction** ... A sequence of **Orchard-ZSA** Action...

File: zip-0230.rst [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``64 * nActionsOrchard``            |``vSpendAuthSigsOrchard`` |``byte[64 * nActionsOrchard]``          |Authorizing signatures for each ZSA Orchard Action.                        |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+|``64``                              |``bindingSigOrchard``     |``byte[64]``                            |An Orchard binding signature on the SIGHASH transaction hash.              |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+| **ZSA Burn Fields**                                                                                                                                                                |
```
Comment (PaulLaux) [inline zip-0230.rst:171]: Orchard ZSA Burn

File: zip-0230.rst [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+| ``varies``                         | ``nAssetBurn``           | ``compactSize``                        | The number of Assets burnt.                                               |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+| ``40 * nAssetBurn``                | ``vAssetBurn``           | ``AssetBurn[nAssetBurn]``              | A sequence of Asset Burn descriptions,                                    |
+|                                    |                          |                                        | encoded per `ZSA Asset Burn Description`_.                                |
++------------------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------------+
+| **ZSA Issuance Fields**                                                                                                                                                            |
```
Comment (PaulLaux) [inline zip-0230.rst:178]: Orchard ZSA Iss..

File: zip-0230.rst [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
++-----------------------------+--------------------------+--------------------------------------+------------------------------------------------------------+
+|``80``                       |``outCiphertext``         |``byte[80]``                          |The encrypted contents of the byte string created by        |
+|                             |                          |                                      |concatenation of the transmission key with the ephemeral    |
+|                             |                          |                                      |secret key.                                                 |
++-----------------------------+--------------------------+--------------------------------------+------------------------------------------------------------+
+
+The encodings of each of these elements are defined in §7.4 ‘Output Description Encoding
+and Consensus’ of the Zcash Protocol Specification [#protocol-outputdesc]_.
+
+ZSA Orchard Action Description (``ZSAOrchardAction``)
```
Comment (PaulLaux) [inline zip-0230.rst:265]: Orchard ZSA

File: zip-0230.rst [hunk 7]
```diff
@@ -0,0 +1,352 @@
...
++-----------------------------+--------------------------+--------------------------------------+------------------------------------------------------------+
+|``80``                       |``outCiphertext``         |``byte[80]``                          |The encrypted contents of the byte string created by        |
+|                             |                          |                                      |concatenation of the transmission key with the ephemeral    |
+|                             |                          |                                      |secret key.                                                 |
++-----------------------------+--------------------------+--------------------------------------+------------------------------------------------------------+
+
+The encodings of each of these elements are defined in §7.5 ‘Action Description Encoding
+and Consensus’ of the Zcash Protocol Specification [#protocol-actiondesc]_.
+
+ZSA Asset Burn Description
```
Comment (PaulLaux) [inline zip-0230.rst:293]: Orchard ZSA

File: README.rst [hunk 8]
```diff
@@ -37,7 +37,7 @@ Use ``make`` to check that you are using correct
 `reStructuredText <https://docutils.sourceforge.io/rst.html>`__ or
 `Markdown <https://pandoc.org/MANUAL.html#pandocs-markdown>`__ syntax,
 and double-check the generated ``draft-*.html`` file before filing a Pull Request.
-See `here <protocol/README.rst>`__ for the project dependencies.
+
```
Comment (PaulLaux) [inline README.rst]: Should not be removed

Comment (PaulLaux) [reply README.rst]: Or should it?

File: zip-0226.rst [hunk 9]
```diff
@@ -315,88 +315,8 @@ The input note in the old note commitment integrity check must either include an
 ZSA Transaction Structure
```
Comment (PaulLaux) [inline zip-0226.rst:315]: Orchard ZSA

File: zip-0230.rst (hunk 3 above)
Comment (vivek-arte) [reply zip-0230.rst:148]: Done

File: zip-0230.rst (hunk 4 above)
Comment (vivek-arte) [reply zip-0230.rst:171]: Done

File: zip-0230.rst (hunk 5 above)
Comment (vivek-arte) [reply zip-0230.rst:178]: Done

File: zip-0230.rst (hunk 6 above)
Comment (vivek-arte) [reply zip-0230.rst:265]: Done

File: zip-0230.rst (hunk 7 above)
Comment (vivek-arte) [reply zip-0230.rst:293]: Done

File: zip-0230.rst [hunk 10]
```diff
@@ -0,0 +1,352 @@
...
+--------------------------
+
+A ZSA Asset Burn description is encoded in a transaction as an instance of an ``AssetBurn`` type:
```
[the remark this answers is not in the document]
Comment (vivek-arte) [reply zip-0230.rst:296]: Done

File: zip-0226.rst (hunk 9 above)
Comment (vivek-arte) [reply zip-0226.rst:315]: Done

File: README.rst (hunk 8 above)
Comment (vivek-arte) [reply README.rst]: Added it back, it was added by Antoine some months ago.

File: index.html [hunk 11]
```diff
@@ -49,92 +49,95 @@
...
+  <tr> <td><strike>1009</strike></td> <td class="left"><strike><a href="zip-1009">   Five-Entity Strategic Council</a></strike></td> <td>   Obsolete</td>
+  <tr> <td><strike>1010</strike></td> <td class="left"><strike><a href="zip-1010">   Compromise Dev Fund Proposal With Diverse Funding Streams</a></strike></td> <td>   Obsolete</td>
+  <tr> <td><strike>1011</strike></td> <td class="left"><strike><a href="zip-1011">   Decentralize the Dev Fee</a></strike></td> <td>   Obsolete</td>
+  <tr> <td><strike>1012</strike></td> <td class="left"><strike><a href="zip-1012">   Dev Fund to ECC + ZF + Major Grants</a></strike></td> <td>   Obsolete</td>
+  <tr> <td><strike>1013</strike></td> <td class="left"><strike><a href="zip-1013">   Keep It Simple, Zcashers: 10% to ECC, 10% to ZF</a></strike></td> <td>   Obsolete</td>
+  <tr> <td>1014</td> <td class="left"><a href="zip-1014">   Establishing a Dev Fund for ECC, ZF, and Major Grants</a></td> <td>   Active</td>
+  <tr> <td>guide</td> <td class="left"><a href="zip-guide">   {Something Short and To the Point}</a></td> <td>   Draft</td>
 </table></embed></section>
```
[the remark this answers is not in the document]
Comment (vivek-arte) [reply index.html]: Fixed this. When I run `make` the rst2html5 adds additional spacing for me, that's what was making this weird.

File: zip-0230.rst [hunk 12]
```diff
@@ -0,0 +1,352 @@
...
+
+This ZIP also depends upon and defines modifications to the computation of the values
+**TxId Digest**, **Signature Digest**, and **Authorizing Data Commitment** defined by ZIP
+244 [#zip-0244]_.
+
+
+Motivation
+==========
+
+The new Orchard shielded pool requires serialized data elements that are distinct from
```
Comment (PaulLaux) [inline zip-0230.rst:46]: The Orchard-ZSA protocol requires...

File: zip-0230.rst [hunk 13]
```diff
@@ -0,0 +1,352 @@
...
+RFC 2119. [#RFC2119]_
+
+The character § is used when referring to sections of the Zcash Protocol Specification
+[#protocol]_.
+
+
+Abstract
+========
+
+This proposal defines a new Zcash peer-to-peer transaction format, which includes data that supports Zcash Shielded Assets (ZSAs) on the Orchard shielded pool [#protocol]_.  The
```
Comment (PaulLaux) [inline zip-0230.rst:33]: "data that supports the the Orchard-ZSA protocol and all Zcash Shielded Assets (ZSAs) related operations."

File: zip-0230.rst [hunk 14]
```diff
@@ -0,0 +1,352 @@
...
+any previous Zcash transaction. Since ZIP 244 was activated in NU5, the
+v5 and later serialized transaction formats are not consensus-critical. 
+Thus, this ZIP defines format that can easily accommodate future extensions,
+where elements or a given pool are kept separate.
+
+
+Requirements
+============
+
+The new format must fully support the Orchard protocol.
```
Comment (PaulLaux) [inline zip-0230.rst:56]: Orchard-ZSA protocol

File: zip-0230.rst [hunk 15]
```diff
@@ -0,0 +1,352 @@
...
+
+The new format must fully support Zcash Shielded Assets (ZSAs).
```
Comment (PaulLaux) [inline zip-0230.rst:58]: not required

**Review by PaulLaux — APPROVED** [review]
Nice!
Approved with minor comments.

File: zip-0230.rst (hunk 12 above)
Comment (vivek-arte) [reply zip-0230.rst:46]: Done

File: zip-0230.rst (hunk 13 above)
Comment (vivek-arte) [reply zip-0230.rst:33]: Done.

File: zip-0230.rst (hunk 14 above)
Comment (vivek-arte) [reply zip-0230.rst:56]: Done

File: zip-0230.rst (hunk 15 above)
Comment (vivek-arte) [reply zip-0230.rst:58]: Done

**Pull request merged**